.circleci/config.yml

```diff
@@ -1,13 +1,28 @@
 version: 2
 jobs:
   build:
-    machine: true
+    docker:
+      - image: mhart/alpine-node:8.9.4
```

Comment:
This is a different alpine version from Hilary, should we update that too?

Reply:
Well, hilary docker is still on ubuntu, isn't it? I remember I tried to migrate it to alpine at some point, but there was something blocking it, can't recall what exactly.

I don't think there is a problem though, because this is just to make sure 3akai-ux builds correctly so it's pretty agnostic.